### ISSUE TITLE
Backup breaks when WRAPPER_CONF_DIR is not set

### ISSUE THREAD

**Issue by dAnjou** [gocd/gocd#8525]
##### Issue Type

- Bug Report

##### Summary

Without having the environment variable WRAPPER_CONF_DIR set, the backup via admin interface breaks.

We've noticed this while upgrading our 19.6.0 deployment to 20.1.0. The relevant commit is https://github.com/gocd/gocd/pull/7312/commits/7768b20842fe8c3adc3f0ce7d8cd98475a5926ce and the issue triggering the change is https://github.com/gocd/gocd/issues/7301.

###### Basic environment details

* Go Version: 20.1.0, but looking at the code it seems the behavior has not changed in version 20.7.0
* JAVA Version: 11.0.8
* OS: Amazon Linux 2

##### Steps to Reproduce

1. Deploy GoCD server 
2. Perform backup via UI `/go/admin/backup`

##### Expected Results

Backup succeeds

##### Actual Results

Backup fails at step `Backing up Wrapper Configuration` with error message `Could not find wrapper-config directory`.

##### Possible Fix

We worked around it by setting the environment variable to [the suggested directory for our installation](https://docs.gocd.org/current/installation/install/server/linux.html#location-of-gocd-server-files).

**Comment by maheshp** [comment]
@dAnjou the environment variable `WRAPPER_CONF_DIR` is set by default by Tanuki the Java Service Wrapper used by GoCD. I verified backups on `v20.1.0` and `v20.7.0` using a zip installer and it works as expected.

Which installer are you using for you GoCD server?  Can you share the output of the API support logs, it can be fetched by opening - `http://<go-server-host>/go/api/support`. We don't need the entire output, I am interested in the `Environment Variables` set.

**Comment by dAnjou** [comment]
Hi @maheshp, thanks for the response.

I don't feel comfortable posting all the environment variables here, there are over 1600 and some might contain sensitive information. We've checked `/go/api/support` while debugging the issue and `WRAPPER_CONF_DIR` wasn't set, after we've set it manually it showed up in that list.

We're installing GoCD on Amazon Linux 2, here's a snippet from our Dockerfile:
```
curl https://download.gocd.org/gocd.repo -o /etc/yum.repos.d/gocd.repo && \
  yum -y install go-server-${GOCD_PACKAGE_VERSION}-${GOCD_PACKAGE_BUILD_NUMBER} 
```
And then we're starting the server like this:
```
java -XX:+HeapDumpOnOutOfMemoryError
    -XX:HeapDumpPath=${GO_SERVER_WORKDIR}/artifacts/heapdump.bin
    -Duser.language=${LANGUAGE}
    -Djruby.rack.request.size.threshold.bytes=30000000
    -Duser.country=${COUNTRY}
    -Dcruise.config.dir=${GO_CONFIG_DIR}
    -Dcruise.config.file=${GO_CONFIG_DIR}/cruise-config.xml
    -Dcruise.server.port=${GO_SERVER_PORT}
    -Dcruise.server.ssl.port=${GO_SERVER_SSL_PORT}
    -Dgo.config.repo.gc.periodic=Y
    -Dgo.config.repo.gc.check.interval=3600000
    -Dgocd.log.system.properties.on.startup=Y
    -Dgo.config.repo.gc.cron='25 25 * ? * *'
    -Dgo.config.repo.gc.warning.looseobject.threshold=5000
    -Dgo.config.repo.gc.expire=12
    -Dgo.config.repo.gc.aggressive=N
    -Dgo.database.provider=com.thoughtworks.go.postgresql.PostgresqlDatabase
    -Dhttp.nonProxyHosts=${NO_PROXY}
    -server -jar ${GO_SERVER_BOOT}/lib/go.jar
```
This variable seems to be very optional. I feel like performing a backup shouldn't break when it's not set.

**Comment by maheshp** [comment]
Hello @dAnjou, looking at the way you start your server I understand why the `WRAPPER_CONF_DIR` is not set.
I think it is ok to make the wrapper config backup optional, there is a PR #8539 for the same.

**Comment by kritika-singh3** [comment]
Fixed and verified in #8539
